src/containers/pages/FocusInvestigation/map/active/index.tsx

```diff
@@ -436,12 +436,7 @@ const mapStateToProps = (state: Partial<Store>, ownProps: any) => {
     goals,
     jurisdiction,
     plan,
-    plansArray: getPlansArray(
-      state,
-      InterventionType.FI,
-      [PlanStatus.ACTIVE, PlanStatus.COMPLETE],
-      null
```

Comment:
What was the reason for removing this `null`?

Reply:
`getPlansArray` sets it to null by default. I think its repetition.